src/client/admin/admin.proto

```diff
@@ -17,6 +17,15 @@ import "client/admin/v1_11/pfs/pfs.proto";
 import "client/admin/v1_11/pps/pps.proto";
 import "client/pfs/pfs.proto";
 import "client/pps/pps.proto";
+import "client/auth/auth.proto";
+import "client/enterprise/enterprise.proto";
+
+// CheckAuthToken is the first message in an extract on a cluster
+// where auth is enabled. It verifies that the user provided the correct
+// auth token for the restore before we start restoring. 
+message CheckAuthToken {
```

Comment:
Because we restore auth relatively late in the restore process, the first Op on a cluster with auth is this placeholder that verifies the auth token in the user's context matches the hash used for Activate later on.